frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.test.tsx

```diff
@@ -37,124 +36,89 @@ const mockBpmnContextValue: BpmnContextProps = {
   setDataTasksRemoved: jest.fn(),
 };
 
-jest.mock('app-shared/utils/featureToggleUtils', () => ({
-  shouldDisplayFeature: jest.fn().mockReturnValue(false),
-}));
-
 describe('ConfigPanel', () => {
-  afterEach(jest.clearAllMocks);
-
-  it('should render without crashing', () => {
-    render({ appLibVersion: mockAppLibVersion7, bpmnDetails: null, isEditAllowed: false });
-    expect(
-      screen.getByText(textMock('process_editor.configuration_panel_no_task')),
-    ).toBeInTheDocument();
+  beforeEach(() => {
+    jest.resetAllMocks();
   });
 
-  it('should display the message about selecting a task when bpmnDetails is "null"', () => {
-    render({ bpmnDetails: null });
-    expect(
-      screen.getByText(textMock('process_editor.configuration_panel_no_task')),
-    ).toBeInTheDocument();
-  });
+  it('should render no selected task message', () => {
+    renderConfigPanel({ bpmnDetails: null });
+    const title = screen.getByRole('heading', {
+      name: textMock('process_editor.configuration_panel_no_task_title'),
+    });
 
-  it('should display the message about selecting a task when bpmnDetails.type is "Process"', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.Process } });
-    expect(
-      screen.getByText(textMock('process_editor.configuration_panel_no_task')),
-    ).toBeInTheDocument();
-  });
+    const message = screen.getByText(
+      textMock('process_editor.configuration_panel_no_task_message'),
+    );
 
-  it('should display the message about selected element not being supported when bpmnDetails.type "SequenceFlow"', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.SequenceFlow } });
-    expect(
-      screen.getByText(textMock('process_editor.configuration_panel_element_not_supported')),
-    ).toBeInTheDocument();
+    expect(title).toBeInTheDocument();
+    expect(message).toBeInTheDocument();
   });
 
-  it('should display the message about selected element not being supported when bpmnDetails.type "StartEvent"', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.StartEvent } });
-    expect(
-      screen.getByText(textMock('process_editor.configuration_panel_element_not_supported')),
-    ).toBeInTheDocument();
+  it('should render no supported element message', () => {
+    renderConfigPanel({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.SequenceFlow } });
+    const title = screen.getByRole('heading', {
+      name: textMock('process_editor.configuration_panel_element_not_supported_title'),
+    });
+    const message = screen.getByText(
+      textMock('process_editor.configuration_panel_element_not_supported_message'),
+    );
+    expect(title).toBeInTheDocument();
+    expect(message).toBeInTheDocument();
```

Comment:
Is this test the same as one of the cases in the last `it.each` test? 🤔

Reply:
Oh, snap! It was supposed to be removed when I implemented the `it.each`, great catch!